Demo/FluentUIDemo/ContentView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import FluentUI
+
+struct ContentView: View {
+#if os(macOS)
+    // Until we have a SwiftUI button on macOS, this will have to do.
+    private struct ButtonRepresentable: NSViewRepresentable {
```

Comment:
no macOS swiftui button but we could use notification if we want to rip out the view representable

Reply:
Yeah I totally forgot about that component! If you're okay with it, maybe I'll save that for the first real demo page once we add a proper navigation flow to the app?